crates/driver/src/domain/quote.rs

```diff
@@ -93,8 +95,12 @@ impl Order {
     }
 
     fn fake_auction(&self, gas_price: eth::GasPrice) -> competition::Auction {
+        let random_id = rand::thread_rng()
+            .gen_range(auction::Id::MIN..=auction::Id::MAX)
+            .try_into()
+            .unwrap();
         competition::Auction {
-            id: None,
+            id: random_id,
```

Comment:
Instead of a random one, why not a `0` ID. This can be useful for solvers to disambiguate between quotes and auctions internally (and not include PMMs in quotes for example).